### PR TITLE
Add  cluster_new.yaml, which is cluster.yaml without jinja templates

### PR DESCRIPTION
**Pull request by ebondare**
For my test case I need a file similar to cluster.yaml but without jinja templates, so I added cluster_new.yaml, as we have cephfilesystem.yaml and cephfilesystem_new.yaml.

### CODE REVIEW

File: templates/ocs-deployment/cluster_new.yaml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#################################################################################################################
```
Comment (ebenahar) [inline templates/ocs-deployment/cluster_new.yaml]: @vasukulkarni @ebondare shouldn't we move this under CSI dir and call it cluster.py?

Comment (vasukulkarni) [reply templates/ocs-deployment/cluster_new.yaml]: @ebondare how are you using this one? Are you modifying config and reloading without jinja? If so I think the best place is CSI/ folder

Comment (vasukulkarni) [reply templates/ocs-deployment/cluster_new.yaml]: @ebondare ?

Comment (vasukulkarni) [reply templates/ocs-deployment/cluster_new.yaml]: @petr-balogh we will have to use this in deployment only then its going to be useful?

Comment (ebondare) [reply templates/ocs-deployment/cluster_new.yaml]: > @ebondare how are you using this one? Are you modifying config and reloading without jinja? If so I think the best place is CSI/ folder

As a matter of fact, the testcase I needed this file for was taken from me, so I don't need it anymore. I'll just close this pull request, right?

Comment (petr-balogh) [reply templates/ocs-deployment/cluster_new.yaml]: Yep, please close, if we will start using files without templates in deployment we will update our file or update. Thanks

**Pull request closed without merging**